File: payment-service/src/main/java/com/hedvig/paymentservice/domain/trustlyOrder/commands/PaymentResponseReceivedCommand.java
package com.hedvig.paymentservice.domain.trustlyOrder.commands;

import java.util.UUID;
import org.axonframework.commandhandling.TargetAggregateIdentifier;

import lombok.Value;

@Value
public class PaymentResponseReceivedCommand {
    @TargetAggregateIdentifier
    UUID hedvigOrderId;

    String url;
    String trustlyOrderId;
}

File: payment-service/src/main/java/com/hedvig/paymentservice/domain/trustlyOrder/commands/PaymentResponseReceivedCommand.kt
package com.hedvig.paymentservice.domain.trustlyOrder.commands

import java.util.*
import org.axonframework.commandhandling.TargetAggregateIdentifier

data class PaymentResponseReceivedCommand(

    @TargetAggregateIdentifier  
    val hedvigOrderId: UUID,
    val url: String?,
    val trustlyOrderId: String
)
